docker-compose.yaml

```diff
@@ -0,0 +1,70 @@
+version: "2"
+
+services:
+  kaufmann:
+    build:
+      context: .
+    environment: 
+      KAFKA_HOST: kafka
+      KAFKA_TOPIC: rapids
+      CONSUMER_GROUP: test-consumer-group
+      SCHEMA_REGISTRY_PATH: http://schema_registry:8081/
+      SERVICE_NAME: TEST
+      HOST_NAME: docker-compose-test
+    depends_on: 
+      - kafka
+      - schema_registry
+    volumes:
+      - .:/app/kaufmann
+    command: ["/bin/bash"]
+
+  etcd:
```

Comment:
why do you use etcd here? wouldn't it be easier to go with kafka image containing zookeeper already like https://github.com/spotify/docker-kafka?